### DISCUSSION TITLE
What's sample request body for GetStatus, StartInstance, and Stop Instance?

### DISCUSSION THREAD

**Discussion by DavinY** [bervProject/go-microservice-boilerplate#75] [Q&A]
can you give me example for that Body request

**Comment by berviantoleo** [comment]
Do you mean the REST API provided by the Go API? I think currently hard-coded to turn on/off my EC2 Instances. I can help you to update the code.

**Reply by DavinY** [reply]
I have review your code, it turns out that you are using the environment variable in Local, and it works, thank you for the code reference. just export the env variables and run it,
btw I have another question, can we execute command prompt with aws-cli?

**Reply by berviantoleo** [reply]
I'm not sure about the command prompt that you refer to. But, yes, you can use aws-cli to turn on the ec2 instance. Example command:

`aws start-instances --instance-ids <instance-id>`

**Reply by DavinY** [reply]
I mean, I want to execute this command `expoexport COMMANDLINE_ARGS="--share --enable-insecure-extension-access"
./webui.sh` on the instance terminal with aws-cli after the instance is started
